### PR TITLE
New degree meta

### PR DESCRIPTION
**Pull request by jmbarne3**
**Description**
Adds a number of functions for displaying degree outcomes, projections, news, and career paths. The order and specific layout within the degree template are there only for demonstration, so no asking for me to update styles on this PR! 😛 

**Motivation and Context**
We have all this delicious data in the kitchen but none of the table. Git'r'done!

**How Has This Been Tested?**
Changes are available in dev.

**Types of changes**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

**Checklist:**

- [x] My code follows the code style of this project.
- [ ] My change requires an update to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: dev/acf-export.json [hunk 1]
```diff
@@ -965,7 +1035,7 @@
                 "label": "Projected New Positions Percentage",
                 "name": "degree_prj_change_percentage",
                 "type": "read_only",
-                "instructions": "The projected change in the number of positions available between the start and end year as a percentage.",
+                "instructions": "The projected change in the number of positions available between the start and end year as a precentage.",
```
Comment (cjg89) [inline dev/acf-export.json]: Typo "precentage"

File: dev/acf-export.json [hunk 2]
```diff
@@ -800,6 +816,41 @@
...
+                "instructions": "Determines if degree outcomes are displayed",
+                "required": 0,
+                "conditional_logic": 0,
+                "wrapper": {
+                    "width": "",
+                    "class": "",
+                    "id": ""
+                },
+                "message": "",
+                "default_value": 1,
```
Comment (cadie) [inline dev/acf-export.json]: Talked to Roger about this on Friday and forgot to mention it, but he wants to default to not showing these imported data fields for now.

File: dev/acf-export.json [hunk 3]
```diff
@@ -880,12 +931,31 @@
...
+                "instructions": "Determines if projection data is displayed.",
+                "required": 0,
+                "conditional_logic": 0,
+                "wrapper": {
+                    "width": "",
+                    "class": "",
+                    "id": ""
+                },
+                "message": "",
+                "default_value": 1,
```
Comment (cadie) [inline dev/acf-export.json]: See above comment, default to false.

File: dev/acf-export.json [hunk 4]
```diff
@@ -1026,6 +1150,25 @@
...
+                "instructions": "Determines if career paths are displayed for this degree.",
+                "required": 0,
+                "conditional_logic": 0,
+                "wrapper": {
+                    "width": "",
+                    "class": "",
+                    "id": ""
+                },
+                "message": "",
+                "default_value": 1,
```
Comment (cadie) [inline dev/acf-export.json]: See above comment, default to false.

**Pull request merged**